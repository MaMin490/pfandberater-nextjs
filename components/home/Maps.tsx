import React from 'react';
import styled from 'styled-components';

import { theme } from '../../public/global/colors';
import ChooseCity from './ChooseCity';
const { colors } = theme;

const MapContainer = styled.div`
  position: absolute;
  top: 50%;
  left: 50%;
  color: ${colors.menuGrey};
  transform: translate(-50%, 8rem);
`;

export default function Maps(): JSX.Element {
  return (
    <MapContainer>
      <ChooseCity />
    </MapContainer>
  );
}
