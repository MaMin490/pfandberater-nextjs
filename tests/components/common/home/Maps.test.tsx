import React from 'react';
import { shallow } from 'enzyme';

import Maps from '../../../../components/home/Maps';

describe('Maps', () => {
  it('should render correctly', () => {
    const component = shallow(<Maps />);

    expect(component).toMatchSnapshot();
  });
});
