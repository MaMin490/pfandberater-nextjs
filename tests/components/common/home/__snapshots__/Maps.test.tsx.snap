// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maps should render correctly 1`] = `
<styled.div>
  <ChooseCity />
</styled.div>
`;
